Wirtualnik.Frontend/components/common/PopupMenu.vue

```diff
@@ -96,58 +100,74 @@ export default class PopupMenu extends Vue {
         return event;
     }
 
+    logout() {
+        this.$store.commit(`auth/${TokenMutations.SET_TOKEN}`, '');
+    }
 
-    logout(){
-      this.$store.commit(`auth/${TokenMutations.SET_TOKEN}`, '')
-
+    login() {
+        var token = '';
+        var that = this;
+        console.log('login');
+        this.showAuthWindow({
+            path: 'https://api.zlcn.pro/auth/login/Facebook',
+            callback: function (c: string) {
```

Comment:
Use arrow function instead of explicitly declaring JS function:
`callback: (c: string): void => {...}`

Rename `c` parameter to something more descriptive